vanilla/src/modlauncher/java/org/spongepowered/vanilla/modlauncher/bootstrap/ClientDevLaunchHandler.java

```diff
@@ -41,7 +41,7 @@ public String name() {
 
     @Override
     protected void launchService0(final String[] arguments, final ITransformingClassLoader launchClassLoader) throws Exception {
-        final PluginEnvironment launchPluginEnvironment = Main.getLaunchPluginEnvironment();
+        final PluginEnvironment launchPluginEnvironment = SpongeLaunch.pluginEnvironment;
```

Comment:
Good example here, practice OOP and pass off that plugin environment from Main to the launch method of Client/ServerLauncher.